test/test_OstreamInserters.cpp

```diff
@@ -0,0 +1,223 @@
+#include <gtest/gtest.h>
+
+#include "ConfigFile.hpp"
+#include "LogOstreamInserters.hpp"
```

Comment:
Header file is missing. Leads to failed test compilation.

Reply:
Yes. This was after the last refactor, where I deleted the "centralised" header. 

Added missing headers, now test compiles and runs through